except_star.md

```diff
@@ -835,6 +836,22 @@ Programs will only be impacted by the changes proposed in this PEP once they
 begin to use `ExceptionGroup`s and `except*`.
 
 
+Once programs begin to use these features, there will be migration issues to
+consider:
+
+* An `except Exception:` clause will not catch `ExceptionGroup`s because they
+are derived from `BaseException`. Any such clause will need to be replaced
+by `except (Exception, ExceptionGroup):` or `except *Exception:`.
+
+* Similarly, any `except T:` clause that wraps code which is now potentially
+raising `ExceptionGroup` needs to become `except *T:`, and its body may need
+to be updated.
+
+* Libraries that need to support older python versions will not be able to use
+`except*`, and may need to handle `ExceptionGroup`s raised in user code through
+the low-low_level `ExceptionGroup` APIs.
```

Comment:
I think it's stronger. They may not *raise* `ExceptionGroup`.

Reply:
I think what @1st1  meant with this was that you can:

```
try:
    userCode()
except BaseException as e:
    if python_version >= 3.11:
           if it's an exception group: do stuff to it
    do whatever else
```

Reply:
Otherwise what is there to say? "if you need to support older python versions then you can't use the new features" ?

Reply:
Sorry, I didn't notice this bullet is specifically about older Python versions.

And when I think about the case I was concerned about more, that is just about calling out that raising ExceptionGroup from a library API may break applications that use that API and should thus be considered a new feature of the library. So I think we're good.